monitor/api/models/schema.py

```diff
@@ -2,20 +2,30 @@
 from pydantic import BaseModel, validator
 
 
-class ModelSchema(BaseModel):
+class NodeSchema(BaseModel):
     name: str
     unique_id: str
-    database_name: str
-    schema_name: str
-    table_name: str
     owners: Optional[str]
     tags: Optional[str]
     package_name: str
     description: Optional[str]
     full_path: str
 
 
-class NormalizedModelSchema(ModelSchema):
+class ModelSchema(NodeSchema):
+    database_name: str
+    schema_name: str
+    table_name: str
+
+
+class ExposureSchema(NodeSchema):
```

Comment:
all of the attributes below are mandatory? I think most of them are optional no?

Reply:
I compares it with dbt documentation.
What ever is optional I put as optional, everything else should be mandatory:)

Reply:
Tested the fields and updated